app/models/reference_feature_metrics.rb

```diff
@@ -59,6 +59,6 @@ def time_to_get_for(application)
     provided_at_times = times.map(&:second).compact
     return nil if requested_at_times.blank? || provided_at_times.blank?
 
-    (provided_at_times.min - requested_at_times.max).to_i / 1.day
+    (provided_at_times.max - requested_at_times.min) / 1.day
```

Comment:
This is 100% on me. I noticed this when i reviewed the last PR and then got distracted before i left a comment. Sorry about that!